### PR TITLE
replace runningSince to timeStamp

### PR DESCRIPTION
**Pull request by ArshErgon**
issue  #158 have changed `runningService` to `timeStamp`

### CODE REVIEW

**Review by Kalovelo — CHANGES_REQUESTED** [review]
Hello @ArshErgon , thanks for your contribution! I updated the title of the issue to give a little more context.

The goal is to update the value of `runningSince` from a date string to a timestamp value.

For example


`runningSince: "2022-09-22T10:57:26.614Z"`
should be
`runningSince: 1663807927000`

**Comment by ArshErgon** [comment]
Hey @Kalovelo I apology for the wrong PR, I think `Date.parse(new Date("2022-08-04T04:02:10.909Z"))` will help but how can I use it inside of enum, I thinking about it.

**Comment by Kalovelo** [comment]
> Hey @Kalovelo I apology for the wrong PR, I think `Date.parse(new Date("2022-08-04T04:02:10.909Z"))` will help but how can I use it inside of enum, I thinking about it.

Currently, the type of `ServiceStatus::runningSince` is `Date`. Initially this needs to be updated to `number`, since timestamps are of number type. Then, whenever its value is assigned (see https://github.com/aeternity/state-channel-demo/blob/develop/server/src/services/bot/bot.service.ts#L38), we can use `Date.now()` to get the current timestamp.

:bulb: Keep in mind that corresponding tests will need to be updated as well!

**Comment by ArshErgon** [comment]
I think `new Date().getTime()` or `Date().getTime()` will work.

**Comment by ArshErgon** [comment]
hey @Kalovelo all the checks got passed but I got an error which I never ever had.
```
input: change date to number
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint

```
I just did `git commit -m 'my_msg'`

**Comment by Kalovelo** [comment]
> hey @Kalovelo all the checks got passed but I got an error which I never ever had.
> 
> ```
> input: change date to number
> ✖   subject may not be empty [subject-empty]
> ✖   type may not be empty [type-empty]
> 
> ✖   found 2 problems, 0 warnings
> ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
> ```
> 
> I just did `git commit -m 'my_msg'`

Hello @ArshErgon ! We're about to merge `contributing.md` file, sorry for its absence. Git commit messages follow this approach: https://www.conventionalcommits.org/en/v1.0.0/

**Comment by ArshErgon** [comment]
hi @kenodressel I somehow done with the commit, please view my code

**Comment by Kalovelo** [comment]
> hi @kenodressel I somehow done with the commit, please view my code

Keno is not a maintainer of this project, you can a request a review from either me or @martinkaintas. 

Also please update the code of the current PR instead, thanks!

**Comment by ArshErgon** [comment]
@Kalovelo finally, I think..

**Comment by Kalovelo** [comment]
> @Kalovelo finally, I think..

@ArshErgon 

You can ignore the staging pipelines for now, but tests are failing as well. 
They need to be updated too

**Comment by Kalovelo** [comment]
@ArshErgon Also I still propose using `Date.now()` instead of `new Date()` as the latter generates a new class instance - the `Date.now()` utilizes `Date`'s static method therefore it's more appropriate and also shorter

**Comment by ArshErgon** [comment]
> therefore it's more appropriate and also shorter

can see your vision, but here `server/src/services/bot/bot.interface.ts` L36 I've change the `Date` to `number` and when if I try to remove the `new` inside `server/src/services/bot/bot.service.ts` L38, the code breaks because the `Date` is a `string` and `runningSince` is now a `integer`, will find a way.

**Review by Kalovelo — CHANGES_REQUESTED** [review]
Looks great!

Only the minor nitpicking regarding `Date.now()`, by the time it's updated I will mark it as "hacktoberfest-accepted" and merge it when the event starts :smile:  

*Also don't forget to squash your commits :smiley:

File: server/src/services/bot/bot.service.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ const serviceStatus: ServiceStatus = {
   channelsOpenCurrently: 0,
   channelsInitialized: 0,
   channelsOpened: 0,
-  runningSince: new Date(),
+  runningSince: new Date().getTime(),
```
[the remark this answers is not in the document]
Comment (ArshErgon) [reply server/src/services/bot/bot.service.ts]: thanks, its done

**Pull request merged**